src/components/buttons/primary-button/primary-button.styles.js

```diff
@@ -15,7 +15,7 @@ const getButtonLayoutStyles = size => {
         css`
           border-radius: ${vars.borderRadius4};
           > button {
-            padding: 0 ${vars.spacing8} 0 ${vars.spacing8};
+            padding: 0 ${vars.spacingS} 0 ${vars.spacingS};
```

Comment:
Yeeees!

Reply:
Well I was hoping the hard part of the review could be done by Percy. 😄